{
  "Header": {
    "Original": "CVE-2018-11410 (An issue was discovered in Liblouis 3.5.0. A invalid free in the compi ...)",
    "ID": "CVE-2018-11410",
    "Description": "(An issue was discovered in Liblouis 3.5.0. A invalid free in the compi ...)"
  },
  "Annotations": [
    {
      "Original": "- liblouis 3.5.0-2 (bug #899999)",
      "Type": "package",
      "Package": "liblouis",
      "Kind": "fixed",
      "Version": "3.5.0-2",
      "BugNo": 899999
    },
    {
      "Original": "[stretch] - liblouis 3.0.0-3+deb9u2",
      "Type": "package",
      "Release": "stretch",
      "Package": "liblouis",
      "Kind": "fixed",
      "Version": "3.0.0-3+deb9u2"
    },
    {
      "Original": "[jessie] - liblouis \u003cnot-affected\u003e (Code did not even exist at the time)",
      "Type": "package",
      "Release": "jessie",
      "Package": "liblouis",
      "Kind": "not-affected",
      "Description": "Code did not even exist at the time"
    },
    {
      "Original": "[wheezy] - liblouis \u003cnot-affected\u003e (Code did not even exist at the time)",
      "Type": "package",
      "Release": "wheezy",
      "Package": "liblouis",
      "Kind": "not-affected",
      "Description": "Code did not even exist at the time"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=1582024",
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=1582024"
    },
    {
      "Original": "NOTE: https://github.com/liblouis/liblouis/issues/573",
      "Type": "NOTE",
      "Description": "https://github.com/liblouis/liblouis/issues/573"
    }
  ]
}